go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/filecoin-project/go-jsonrpc v0.7.0
 	github.com/filecoin-project/go-padreader v0.0.1
 	github.com/filecoin-project/go-paramfetch v0.0.4
-	github.com/filecoin-project/go-state-types v0.16.0-rc1
+	github.com/filecoin-project/go-state-types v0.16.0-rc1.0.20241210151506-27c2d581aa19
```

Comment:
We need to bubble up a release before merging this.

Reply:
Looks like the release is already there (rc2). Great. We just need this updated and we are good to go I think 👍